biolink-model.yaml

```diff
@@ -1941,11 +1950,28 @@ slots:
     in_subset:
       - translator_minimal
     mappings:
+      - skos:altLabel
+      - gff3:Alias
+      - alliancegenome:synonyms
       - oboInOwl:hasExactSynonym
       - oboInOwl:hasNarrowSynonym
       - oboInOwl:hasBroadSynonym
       - oboInOwl:hasRelatedSynonym
 
+  xref:
```

Comment:
I assumed we had this already!

Reply:
the mappings model from the blml metamodel is an importable yaml..

this is also the opportunity to do this right. We can have an sssom like model where the mapping is an object, with a type, provenance, comments, etc...